extension/landing-zone/variables.tf

```diff
@@ -0,0 +1,324 @@
+  description = "A list of access tags to apply to the VSI resources created by the module."
+  default     = []
+}
+
+variable "image_name" {
+  description = "Image ID used for VSI. Run 'ibmcloud is images' to find available images. Be aware that region is important for the image since the id's are different in each region."
+  type        = string
+  default     = "ibm-ubuntu-18-04-6-minimal-amd64-2"
+}
+
+variable "machine_type" {
+  description = "VSI machine type"
+  type        = string
+  default     = "cx2-2x4"
+}
+
+variable "security_group" {
+  description = "Security group created for VSI"
+  type = object({
+    name = string
+    rules = list(
+      object({
+        name      = string
+        direction = string
+        source    = string
+        tcp = optional(
+          object({
+            port_max = number
+            port_min = number
+          })
+        )
+        udp = optional(
+          object({
+            port_max = number
+            port_min = number
+          })
+        )
+        icmp = optional(
+          object({
+            type = number
+            code = number
+          })
+        )
+      })
+    )
+  })
+  default = {
+    "name" : "management",
+    "rules" : [
+      {
+        "direction" : "inbound",
+        "name" : "allow-ibm-inbound",
+        "source" : "161.26.0.0/16"
+      },
+      {
+        "direction" : "inbound",
+        "name" : "allow-vpc-inbound",
+        "source" : "10.0.0.0/8"
+      },
+      {
+        "direction" : "outbound",
+        "name" : "allow-vpc-outbound",
+        "source" : "10.0.0.0/8"
+      },
+      {
+        "direction" : "outbound",
+        "name" : "allow-ibm-tcp-53-outbound",
+        "source" : "161.26.0.0/16",
+        "tcp" : {
+          "port_max" : 53,
+          "port_min" : 53
+        }
+      },
+      {
+        "direction" : "outbound",
+        "name" : "allow-ibm-tcp-80-outbound",
+        "source" : "161.26.0.0/16",
+        "tcp" : {
+          "port_max" : 80,
+          "port_min" : 80
+        }
+      },
+      {
+        "direction" : "outbound",
+        "name" : "allow-ibm-tcp-443-outbound",
+        "source" : "161.26.0.0/16",
+        "tcp" : {
+          "port_max" : 443,
+          "port_min" : 443
+        }
+      }
+    ]
+  }
+}
+
+variable "user_data" {
+  description = "User data to initialize VSI deployment"
+  type        = string
+  default     = null
+}
+
+variable "boot_volume_encryption_key" {
+  description = "CRN of boot volume encryption key"
+  type        = string
+}
+
+variable "existing_kms_instance_guid" {
+  description = "The GUID of the Hyper Protect Crypto Services or Key Protect instance in which the key specified in var.kms_key_crn and var.backup_encryption_key_crn is coming from. Required only if var.kms_encryption_enabled is set to true, var.skip_iam_authorization_policy is set to false, and you pass a value for var.kms_key_crn, var.backup_encryption_key_crn, or both."
```

Comment:
this description is wrong. There is no `kms_key_crn` or `backup_encryption_key_crn` variables